components/client-common.rst

```diff
@@ -1,5 +1,3 @@
-.. _client-common:
```

Comment:
with a label like this, you write ``:ref:`client-common```. but to specify a whole chapter, you can do ``:doc:`/path/to/client-common```. i would only use labels when we want to deep link to specific sections.